python/tornado/README.md

```diff
@@ -0,0 +1,20 @@
+# Tornado example
+
+Simple application that creates some traces using Tornado. It uses multiple
+coroutines and synchronous functions with many custom and built-in handlers.
+It also serves static files and has access to a local Redis cache.
+
+## Compatibility stack
+
+* Python 2.7
+* Tornado 4.3
+
+## Backing services
+
+* Redis 3.2
+
+## Getting started
+
+Launch the application
```

Comment:
- Add the compose command to also have Redis running
- Add commands to have a venv with all deps installed in it?